src/hooks/useArticle.js

```diff
@@ -42,3 +47,24 @@ export const useArticle = (lang, title) => {
 
   return [article, loadArticle]
 }
+
+const fixTableCaption = (htmlString, i18n) => {
+  const moreInformationText = i18n.i18n('more-information')

```

Comment:
Ideally, this line should be in the <Table> component since this is building the UI and everything else in this function should be in the transform function in `api/getArticle.js` so it is cached correctly.

Reply:
Initially, I also think it should be in `getArticle`, but then the same reason of having correct language translation for the footer is being built here.

I think it's fine we use i18n in `getArticle` so we can cache the result there.

Reply:
`more-information` is needed only in Article page, not Table.

Reply:
`more-information` is needed only in Article page, not Table.

Reply:
so, I have been trying to use `useI18n()` in `getArticle`, but got error and against the [rules of Only Call Hooks at the Top Level](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level).

another tried will be pass i18n object to `getArticle` which is what I want to avoid when working on Footer section

Opinion?

Reply:
done in 4d9394b1c62d344592347a068b2e0ee8edc62163

_not perfect ☹️ we should think more how we parse the html content in general, including translation and markup, when having more idea_